custom-connector-sdk/src/main/java/com/amazonaws/appflow/custom/connector/model/ErrorCode.java

```diff
@@ -81,5 +81,10 @@
     /**
      * Specifies that the connector encountered failure, for some records, while writing to the application.
      */
-    PartialWriteFailure
+    PartialWriteFailure,
+
+    /**
+     * Specifies that the connector is unable to find the resource like AWS SecretManagerARN etc.
```

Comment:
Specifies that the connector is unable to find resources like AWS SecretManagerARN etc

Reply:
done